v0100/srclib/dpstub.asm

```diff
@@ -884,3 +892,7 @@ regs_len equ $ - regs
 fbuf        resb FBUFSZ ; shared with 32-bit code
```

Comment:
Unrelated to your changes, please add this line
~~~
alignb 16
~~~
immediately before the `fbuf resb` line.

Reply:
Done.